src/EXTRA-MOLECULE/dihedral_nharmonic.cpp

```diff
@@ -90,12 +92,14 @@ void DihedralNHarmonic::compute(int eflag, int vflag)
     vb1y = x[i1][1] - x[i2][1];
     vb1z = x[i1][2] - x[i2][2];
 
+    domain->minimum_image(vb1x, vb1y, vb1z);
```

Comment:
These should be removed, this is not how PBC's are handled in LAMMPS. Instead, it is done in the NTopo classes e.g. `atom1 = domain->closest_image(i,atom1);`